alert-common/src/main/java/com/synopsys/integration/alert/common/action/api/AbstractJobResourceActions.java

```diff
@@ -219,4 +237,27 @@ private boolean checkContextAndDescriptorKey(FieldModel fieldModel, BiFunction<C
         return permissionChecker.apply(configContextEnum, descriptorKey);
     }
 
+    // FIXME More tech debt until we fix the Jobs API
+    private void correctProjectsField(JobFieldModel jobFieldModel) {
+        List<JobProviderProjectFieldModel> projects = jobFieldModel.getConfiguredProviderProjects();
+        if (null == projects) {
+            projects = List.of();
+        }
+
+        String projectFieldKey = "channel.common.configured.project";
```

Comment:
Should this project have access to this static key?

Reply:
No, which is why this is a temporary fix :'(